flake.nix

```diff
@@ -107,16 +116,27 @@
         };
 
         devShells.ci = pkgs.mkShell {
+          nativeBuildInputs = [
+            pkgs.openssl
+            pkgs.rustPlatform.bindgenHook
+            pkgs.zlib
+          ];
           packages = [
+            pkgs.cmake
```

Comment:
I don't exactly know if all of these pkgs are actually required. I've rebuilt a couple of times locally to narrow them down, but had some "non-determinism" gremlins in my machine and it became too time-consuming to figure this out.

Reply:
Looks like openssl went away, yay!